crates/editor_ui/src/hierarchy.rs

```diff
@@ -150,23 +154,30 @@ fn draw_entity<F: ReadOnlyWorldQuery>(
         )
         .show_header(ui, |ui| {
             let mut entity_name = egui::RichText::new(entity_name.clone());
-            if auto_children.get(entity).is_ok() {
+            let is_auto_child = auto_children.get(entity).is_ok();
+            if is_auto_child {
                 entity_name = entity_name.italics();
             }
 
             let response = ui.selectable_label(is_selected, entity_name);
             let is_clicked = response.clicked();
-            response.context_menu(|ui| {
-                hierarchy_entity_context(
-                    ui,
-                    commands,
-                    entity,
-                    changes,
-                    clone_events,
-                    selected,
-                    parent,
-                );
-            });
+            if is_auto_child {
+                response.context_menu(|ui| {
+                    ui.label("Its auto child prefab entity of solid bevy scene. Can't be reparented/deleted. Try \"Unpack gltf as prefab\" for that.");
```

Comment:
The phrasing got a little confusing, Did you mean something like `"Concrete Bevy scene auto child cannot be reparented or deleted. Try \"Unpack gltf as prefab\" for that.".` Also, this feels more like an error toast than a label

Reply:
Having it as an event will be easier to test as well and will be registered in the event window (Top right corner).
`mut toast: EventWriter<ToastMessage>`

`toast.send(ToastMessage::new(Concrete Bevy scene auto child cannot be reparented or deleted. Try \"Unpack gltf as prefab\" for that.", ToastKind::Error)`

Reply:
Its not error xDD Its just reminder that it is not "full" prefab entity and only support part of functionality

Reply:
We could probably add an warning label to that. Can you create an example where this happens so I can discuss with the UX?

Reply:
This message is displayed instead of the context menu when right-clicking an entity in the hierarchy if the entity is an incomplete prefab

Reply:
I would just make the text clearer:

"Concrete Bevy scene cannot be reparented or deleted. Try \"Unpack gltf as prefab\" for that." 

And add a red label with color: ERROR_COLOR

Reply:
Solved https://github.com/rewin123/space_editor/pull/239

Reply:
<img width="263" alt="Captura de Tela 2024-03-05 às 7 49 31 PM" src="https://github.com/rewin123/space_editor/assets/14813660/de84aab6-b15d-4807-9ef2-04ed7541adcb">